web/src/EducationBenchmarking.Web/ViewModels/SchoolPlanningViewModel.cs

```diff
@@ -6,5 +6,7 @@ public class SchoolPlanningViewModel(School school)
 {
```

Comment:
Can this model be removed now we have SchoolPlanViewModel?

Reply:
Good spot - pushed change